exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/Entity.kt

```diff
@@ -54,37 +61,61 @@ open class Entity<ID : Comparable<ID>>(val id: EntityID<ID>) {
+                                                                                                                              reference.referee!!.getValue(
+                                                                                                                                  this,
+                                                                                                                                  desc
+                                                                                                                              ) == refValue
+                                                                                                                          }) { reference.referee<REF>()!! eq refValue }
                     .singleOrNull()
             } ?: error("Cannot find ${factory.table.tableName} WHERE id=$refValue")
         }
     }
 
-    operator fun <REF : Comparable<REF>, RID : Comparable<RID>, T : Entity<RID>> Reference<REF, RID, T>.setValue(o: Entity<ID>, desc: KProperty<*>, value: T) {
+    operator fun <REF : Comparable<REF>, RID : Comparable<RID>, T : Entity<RID>> Reference<REF, RID, T>.setValue(
+        o: Entity<ID>,
+        desc: KProperty<*>,
+        value: T
+    ) {
         if (db != value.db) error("Can't link entities from different databases.")
         value.id.value // flush before creating reference on it
         val refValue = value.run { reference.referee<REF>()!!.getValue(this, desc) }
         reference.setValue(o, desc, refValue)
     }
 
-    operator fun <REF : Comparable<REF>, RID : Comparable<RID>, T : Entity<RID>> OptionalReference<REF, RID, T>.getValue(o: Entity<ID>, desc: KProperty<*>): T? {
+    operator fun <REF : Comparable<REF>, RID : Comparable<RID>, T : Entity<RID>> OptionalReference<REF, RID, T>.getValue(
+        o: Entity<ID>,
+        desc: KProperty<*>
+    ): T? {
         return executeAsPartOfEntityLifecycle {
             val refValue = reference.getValue(o, desc)
             when {
-                refValue == null -> null
+                refValue == null                                                        -> null
                 refValue is EntityID<*> && reference.referee<REF>() == factory.table.id -> factory.findById(refValue.value as RID)
-                else -> factory.findWithCacheCondition({ reference.referee!!.getValue(this, desc) == refValue }) { reference.referee<REF>()!! eq refValue }
+                else                                                                    -> factory.findWithCacheCondition({
+                                                                                                                              reference.referee!!.getValue(
+                                                                                                                                  this,
+                                                                                                                                  desc
```

Comment:
What does the change like this add?
It makes the code card to read, and it introduces merge conflicts. For instance, if I change one branch in `when`, the formatter would force me to edit all the others, thus the diff would be unnecessary big.